cmd/admin-handlers-config-kv.go

```diff
@@ -211,7 +211,12 @@ func setConfigKV(ctx context.Context, objectAPI ObjectLayer, kvBytes []byte) (re
 		return
 	}
 
-	if verr := validateConfig(result.Cfg, result.SubSys); verr != nil {
+	tgts, err := config.ParseConfigTargetID(bytes.NewReader(kvBytes))
```

Comment:
```go
result.SubSys, _, target, err = config.GetSubSys(string(kvBytes))
```

^^ already provides the target which is going to be edited/added. We can use this instead.

Reply:
@Praveenrajmani It used in ParseConfigTargetID